deployments/data102/image/apt.txt

```diff
@@ -0,0 +1,93 @@
+# Some linux packages for basic terminal work, particularly
+# oriented at users new to Unix/cmd line environments.
+
+# Basic unix tools
+man
+man-db
+manpages-posix
+manpages-dev
+manpages-posix-dev
+
+# Download tools
+curl
+wget
+
+# Core text editors on a *nix box: vim and emacs
```

Comment:
I'm pretty sure that most of the editors and terminal file managers are for other courses. (though the comments are so old they don't indicate which)

Reply:
(The comment above was for lines starting at 15)

Reply:
@ryanlovett Removed a bunch of packages in my next commit. Can you please review?